image-filters/image-filters.markdown

```diff
@@ -1552,16 +1551,18 @@ Making things look just so. Making them fail over. Maintaining that 99.9%
 uptime. We spend more time hunting down corner cases than refining algorithms.
```

Comment:
line above - "fail over" - I think someone else changed this, because I had to look up what it means. It's actually "failover" and maybe we should link to a definition?